setup.cfg

```diff
@@ -1,2 +1,14 @@
 [bdist_wheel]
 universal=1
+
+[flake8]
```

Comment:
I am not quite sure about the need of this change

Reply:
The rationale is one less file to deal with. Historically in this project we haven't made much use of setup.cfg. There are tons of configuration elements we can add to setup.cfg and reduce configuration file drift.